core/src/main/scala/org/zalando/grafter/Visualize.scala

```diff
@@ -0,0 +1,107 @@
+package org.zalando.grafter
+
+import scala.util.matching.Regex
+import org.bitbucket.inkytonik.kiama.relation.{Relation, Tree}
+
+trait Visualize {
+
+  type HashCode = Int
+
+  /**
+   * Generate a representation of the graph induced by the rootComponent
+   * in GraphViz DOT format (http://www.graphviz.org)
+   * A components is taken into account if its package is included and not excluded by
+   * the provided regular expressions.
+   */
+  def asDotString[T <: Product](root:       T,
+                                filter:     Product => Boolean = Visualize.packageFilter(),
```

Comment:
As well, but my point is more towards the right side of the `=`: what is included and excluded in this case? Perhaps we could have just another method that calls `packageFilters`, with a more specific name, like `filterAll`, or `excludesAll` or something like that.